template/tsconfig.json

```diff
@@ -1,64 +1,21 @@
 {
-    "compilerOptions": {
-        /* Basic Options */
```

Comment:
I cleaned out the `tsconfig.json`, removing all the commented out options & descriptions. Some of these were rather outdated and I'd rather point users as TS's own docs